positional/src/field.rs

```diff
@@ -2,7 +2,6 @@ use pad::{Alignment, PadStr};
 use std::fmt::Display;
 
 #[doc(hidden)]
```

Comment:
I wasn't sure whether this attribute should apply only to the trait or the whole module, if it's the latter, this line should be changed to `#![doc(hidden)]` and moved to the top of the file.

Reply:
looking at the doc comment below I'm not sure if this trait in particular should be hidden at all